apps/nextjs/src/app/api/chat/protocol.ts

```diff
@@ -5,7 +5,9 @@ import {
 import { StreamingTextResponse } from "ai";
 
 export function streamingJSON(message: ErrorDocument | ActionDocument) {
-  const errorMessage = JSON.stringify(message);
+  const jsonContent = JSON.stringify(message);
+  const errorMessage = `0:"${jsonContent.replace(/"/g, '\\"')}"`;
```

Comment:
Fine for here, and I think this 0: is just a temp measure until we update the client package? Otherwise would thinking would be good for it to be its own util